app/models/green_flag/feature.rb

```diff
@@ -53,6 +53,15 @@ def latest_version
 
     last_rule.present? ? last_rule.version_number : version_number
   end
+
+  def fully_enabled?
+    rules.where(:group_key => "Everyone", :percentage => 100).size > 0
```

Comment:
If there are rules above the "Everyone" rule, the feature might not be fully enabled.  I think this should be something like: `rules.where("percentage < 100").any?`